subsys/usb/class/audio/audio.c

```diff
@@ -689,6 +689,11 @@ static int audio_custom_handler(struct usb_setup_packet *pSetup, int32_t *len,
 
 	uint8_t iface = (pSetup->wIndex) & 0xFF;
 
+	if (REQTYPE_GET_RECIP(pSetup->bmRequestType) ==
+	    REQTYPE_RECIP_INTERFACE) {
+		return -EINVAL;
+	}
+
```

Comment:
Wouldn't all interface request be dropped by `custom_handler` right now? Wouldn't the class be unable to set `rx/tx_enable` flags by this addition?

Reply:
Yes, thanks, that should be inverse, also `!= REQTYPE_RECIP_INTERFACE`.